sys/linux/dev_hidraw.txt

```diff
@@ -0,0 +1,35 @@
+# Copyright 2019 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+include <uapi/linux/hidraw.h>
+
+resource fd_hidraw[fd]
+
+syz_open_dev$hidraw(dev ptr[in, string["/dev/hidraw#"]], id intptr, flags flags[open_flags]) fd_hidraw
+
+write$hidraw(fd fd_hidraw, data buffer[in], len bytesize[data])
```

Comment:
Add a TODO that there is binary format for the written data.

Reply:
Done.